standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -5094,12 +5109,13 @@ private int abortTxns(Connection dbConn, List<Long> txnids,
    * @throws SQLException
    */
   private int abortTxns(Connection dbConn, List<Long> txnids, boolean checkHeartbeat,
-                        boolean skipCount, boolean isReplReplayed)
+                        boolean skipCount, boolean isReplReplayed, TxnErrorMsg errorMsg)
       throws SQLException, MetaException {
     Statement stmt = null;
     if (txnids.isEmpty()) {
       return 0;
     }
+    LOG.debug("Trying to abort txns due to : {}. Aborted Transaction IDs : {}", errorMsg, txnids);
```

Comment:
LOG.debug("Trying to abort txns {} due to {}", txnids, errorMsg) 
see below

Reply:
Implemented as suggested.